.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/FineTuningTests.cs

```diff
@@ -22,7 +22,7 @@ namespace Azure.AI.OpenAI.Tests;
 
 public class FineTuningTests : AoaiTestBase<FineTuningClient>
 {
-    public FineTuningTests(bool isAsync) : base(isAsync)
+    public FineTuningTests(bool isAsync) : base(isAsync, RecordedTestMode.Live)
```

Comment:
I always forget to undo these, but I expect you don't want to commit this to main.

Reply:
good catch. I forgot I did that!